pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -93,11 +98,13 @@ func TestCMServerDefaults(t *testing.T) {
 			LookupCacheSizeForRC:              4096,
 			LookupCacheSizeForRS:              4096,
 			LookupCacheSizeForDaemonSet:       1024,
+			ConfigureCloudRoutes:              true,
+			NodeCIDRMaskSize:                  24,
 			ServiceSyncPeriod:                 unversioned.Duration{Duration: 5 * time.Minute},
 			NodeSyncPeriod:                    unversioned.Duration{Duration: 10 * time.Second},
 			ResourceQuotaSyncPeriod:           unversioned.Duration{Duration: 5 * time.Minute},
 			NamespaceSyncPeriod:               unversioned.Duration{Duration: 5 * time.Minute},
-			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 10 * time.Minute},
+			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 15 * time.Second},
```

Comment:
It's doing the M to N comparison of PVs and PVCs but it's not relisting
(just scanning for deltas that couldn't be satisfied).

On Thu, Jul 7, 2016 at 3:44 PM, David Eads <notifications@github.com> wrote:

> In pkg/cmd/server/kubernetes/master_config_test.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69973443>:
>
> >  			ServiceSyncPeriod:                 unversioned.Duration{Duration: 5 * time.Minute},
> >  			NodeSyncPeriod:                    unversioned.Duration{Duration: 10 * time.Second},
> >  			ResourceQuotaSyncPeriod:           unversioned.Duration{Duration: 5 * time.Minute},
> >  			NamespaceSyncPeriod:               unversioned.Duration{Duration: 5 * time.Minute},
> > -			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 10 * time.Minute},
> > +			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 15 * time.Second},
>
> This is a huge change. Do you know why its so tight now? If not, we should
> have an issue to figure out why. That seems crushing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/b2902157077dc7f476bc028c0cf3ec517871a436..0d6cd9c8e4ca59d63b9d3e70ee6d32af120e7469#r69973443>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p9b3jFSTqQmSSIx5doy_hnOMGNJiks5qTVcBgaJpZM4I-xTE>
> .
>